pages/idt.adoc

```diff
@@ -0,0 +1,142 @@
+---
+title: Interrupt descriptor table
+tags: x86, x64
+category: Interrupts
+description: IDT specification
+---
+:source-language: c
+
+== Interrupt Descriptor Table
```

Comment:
No heading for the entire article, that is done automatically.
Upgrade all section headings by a level.